### ISSUE TITLE
Better multi device support

### ISSUE THREAD

**Issue by UfukUstali** [ClementGre/ThreeFingerDragOnWindows#46]
Currently there is no way to set different speed and acceleration settings for different devices.

Example from my own experince:
When I am on the go and using my laptop by itself the builtin touchpad requires the speed to be 60 to feel the same as when I dock the laptop and use the external Magic Trackpad (with community developed Windows Precision Drivers) which requires the speed to be set to 30.

This is only the case when using the ThreeFingerDrag normal operation of the different devices feels the same without any additional steps.

**Comment by ClementGre** [comment]
You are right, TFDOW already supports switching from one device to another, but there is no way to adjust settings for each device.
This is not a feature in my top priority but I'll look into it in the next months.

**Comment by cunba-ai** [comment]
https://github.com/ClementGre/ThreeFingerDragOnWindows/pull/95

**Comment by ClementGre** [comment]
Fixed by @cunba-ai in v1.0.7.
Some improvements can still be made, though, allowing customization of more options on a per-device basis.
